en/components/grids_templates/keyboard_navigation.md

```diff
@@ -22,35 +22,77 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 
 ## @@igComponent Keyboard Navigation
 
-Keyboard navigation is available by default in any grid and aims at covering as many as possible features and scenarios for the end user. When you focus a specific cell and press one of the following key combinations, the described behavior is performed.
+Keyboard navigation is available by default in the @@igxName and aims at covering as many as possible features and scenarios for the end user. Now in order to improve the overall page navigation experience we reduced the tab stops in the grid. Currently we have only the following main tab stops:
```

Comment:
Keyboard navigation enhances the accessibility of the @@igComponent by providing vast details of every element while navigating through them. This functionality comes with the @@igComponent by the default, although the user has the freedom to override it by implementing a custom one.

The tabulations of the grid have been reduced so that it is fully compliant and convenient to use.
Currently, the grid introduces the following tab stops: